### PR TITLE
ID Localization for Task Index and Example.

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR addressing #20111

### CODE REVIEW

File: content/id/docs/tasks/_index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Tugas-tugas (_Tasks_)
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:2]:
```suggestion
title: Tugas (Tasks)
```
remove italicizing on title

File: content/id/docs/tasks/_index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (_Dashboard_)
+
+Menyebarkan dan mengakses Dashboard antarmuka pengguna berbasis web untuk
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:22]:
```suggestion
Melakukan _deploy_ dan mengakses dasbor berbasis web untuk 
```
to make it easier to understand in Bahasa, wdyt?

File: content/id/docs/tasks/_index.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+Bagian dokumentasi Kubernetes ini berisi halaman-halaman yang perlihatkan 
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (_Dashboard_)
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:20]:
```suggestion
## Antarmuka Pengguna Berbasis Web (Dasbor)
```

File: content/id/docs/tasks/_index.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+Menyebarkan dan mengakses Dashboard antarmuka pengguna berbasis web untuk 
+membantu kamu mengelola dan memantau aplikasi yang dimasukkan ke dalam kontainer
+di Kubernetes.
+
+## Menggunakan Baris Perintah kubectl
+
+Instalasi dan konfigurasi utilitas baris perintah `kubectl` yang digunakan untuk
+mengelola secara langsung klaster Kubernetes.
+
+## Mengkonfigurasi Pod and Kontainer
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:31]:
```suggestion
## Mengkonfigurasi Pod and Container
```

File: content/id/docs/tasks/_index.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+Melakukan tugas konfigurasi yang umum untuk Pod dan Kontainer.
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:33]:
```suggestion
Melakukan tugas konfigurasi yang umum untuk Pod dan Container.
```

File: content/id/docs/tasks/_index.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+## Mengakses Aplikasi dalam Klaster
+
+Mengkonfigurasi _load balancing_, _port forwarding_, atau membangun _firewall_
+atau konfigurasi DNS untuk mengakses aplikasi dalam sebuah klaster.
+
+## _Monitoring, Logging_, dan _Debugging_
+
+Mengatur _monitoring_ (pemantauan) dan _logging_ (pencatatan) untuk memecahkan 
+masalah klaster atau men-_debug_ (melacak) aplikasi yang dikontainerisasi.
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:52]:
```suggestion
masalah klaster atau melakukan _debug_ (melacak) aplikasi yang dikontainerisasi.
```

File: content/id/docs/tasks/example-task-template.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
+---
+title: Contoh _Template_ Tugas (_Task_)
```
Comment (wahyuoi) [inline content/id/docs/tasks/example-task-template.md:2]:
```suggestion
title: Contoh Template Tugas (Task)
```

File: content/id/includes/task-tutorial-prereqs.md [hunk 8]
```diff
@@ -0,0 +1,8 @@
+Kamu harus memiliki klaster Kubernetes, dan perangkat baris perintah `kubectl` 
+juga harus dikonfigurasikan untuk berkomunikasi dengan klaster Anda. Jika kamu
```
Comment (wahyuoi) [inline content/id/includes/task-tutorial-prereqs.md:2]:
```suggestion
juga harus dikonfigurasikan untuk berkomunikasi dengan klaster kamu. Jika kamu 
```

File: content/id/docs/tasks/example-task-template.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Lakukan ini.
+1. Selanjutnya lakukan ini. Bila mungkin silahkan baca [penjelasan terkait](...).
+
+{{% /capture %}}
+
+{{% capture discussion %}}
+
+## Memahami ...
+**[Bagian optional]**
```
Comment (wahyuoi) [inline content/id/docs/tasks/example-task-template.md:37]:
```suggestion
**[Bagian opsional]**
```

File: content/id/docs/tasks/example-task-template.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+Berikut ini hal-hal yang menarik untuk diketahui tentang langkah-langkah yang baru saja kamu lakukan.
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+**[Bagian optional]**
+
+* Pelajari tentang [menulis topik baru](/docs/home/contribute/write-new-topic/).
+* Lihat [menggunakan _template_ halaman - _template_ tugas](/docs/home/contribute/page-templates/#task_template) untuk mengetahui cara menggunakan template ini.
```
Comment (wahyuoi) [inline content/id/docs/tasks/example-task-template.md:48]:
```suggestion
* Lihat [menggunakan _template_ halaman - _template_ tugas](/docs/home/contribute/page-templates/#task_template) untuk mengetahui cara menggunakan _template_ ini.
```

File: content/id/docs/tasks/_index.md (hunk 2 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/_index.md:22]: Ok. But, I would like to change `dasbor` into `dasbor (_dashboard_)` to link with the title.

File: content/id/docs/tasks/_index.md (hunk 4 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/_index.md:31]: I will keep `Kontainer` because most of the other translations already use that.

Comment (wahyuoi) [reply content/id/docs/tasks/_index.md:31]: we use both Kontainer and Container (K8s Object)

this subtitle refer to K8s Object

File: content/id/docs/tasks/_index.md (hunk 5 above)
Comment (wahyuoi) [reply content/id/docs/tasks/_index.md:33]: same as above, refer to K8s Object

File: content/id/docs/tasks/_index.md (hunk 4 above)
Comment (wahyuoi) [reply content/id/docs/tasks/_index.md:31]: @ariscahyadi

File: content/id/docs/tasks/_index.md (hunk 5 above)
Comment (wahyuoi) [reply content/id/docs/tasks/_index.md:33]: @ariscahyadi

File: content/id/docs/tasks/_index.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+
+Bagian dokumentasi Kubernetes ini berisi halaman-halaman yang perlihatkan 
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (Dasbor)
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:20]:
```suggestion
## Antarmuka Pengguna Berbasis Web (Dashboard)
```
Revert back, after discussion with @phanama for other PR, found out that Dashboard here refer to https://github.com/kubernetes/dashboard 
so keep it as is, no need to translate to dasbor

File: content/id/docs/tasks/_index.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+Melakukan _deploy_ dan mengakses dasbor (_dashboard_) berbasis web untuk 
+membantu kamu mengelola dan memantau aplikasi yang dimasukkan ke dalam kontainer
+di Kubernetes.
+
+## Menggunakan Baris Perintah kubectl
+
+Instalasi dan konfigurasi utilitas baris perintah `kubectl` yang digunakan untuk
+mengelola secara langsung klaster Kubernetes.
+
+## Mengkonfigurasi Pod and Kontainer
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:31]:
```suggestion
## Mengkonfigurasi Pod and Container
```
Container here refer to https://kubernetes.io/docs/concepts/containers/
and we tried to change existing Kontainer to Container for k8s object in https://github.com/kubernetes/website/pull/18177
but it kinda tricky, discussed [here on slack channel](https://kubernetes.slack.com/archives/CJ1LUCUHM/p1576345141002200)

File: content/id/docs/tasks/_index.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (Dasbor)
+
+Melakukan _deploy_ dan mengakses dasbor (_dashboard_) berbasis web untuk
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:22]:
```suggestion
Melakukan _deploy_ dan mengakses Dashboard berbasis web untuk 
```
same as above

File: content/id/docs/tasks/_index.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+di Kubernetes.
+
+## Menggunakan Baris Perintah kubectl
+
+Instalasi dan konfigurasi utilitas baris perintah `kubectl` yang digunakan untuk
+mengelola secara langsung klaster Kubernetes.
+
+## Mengkonfigurasi Pod and Kontainer
+
+Melakukan tugas konfigurasi yang umum untuk Pod dan Kontainer.
```
Comment (wahyuoi) [inline content/id/docs/tasks/_index.md:33]:
```suggestion
Melakukan tugas konfigurasi yang umum untuk Pod dan Container.
```

File: content/id/docs/tasks/_index.md (hunk 13 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/_index.md:22]: I think it should lowercase because it does not belong to Kubernetes resource name.

**Comment by girikuncoro** [comment]
All the translated pages under Task section cannot be discovered without this main page, thanks for contributing mas @ariscahyadi ! 🎉 

/lgtm
/approve

**Comment by ariscahyadi** [comment]
> All the translated pages under Task section cannot be discovered without this main page, thanks for contributing mas @ariscahyadi ! 🎉
> 
> /lgtm
> /approve

You're welcome... thanks for the opportunity @girikuncoro ... hope to contribute more :)

**Pull request merged**